homeassistant/components/yale_smart_alarm/config_flow.py

```diff
@@ -107,6 +139,13 @@ async def async_step_user(self, user_input=None):
                     data_schema=DATA_SCHEMA,
                     errors={"base": "invalid_auth"},
                 )
+            except requests.HTTPError as error:
+                LOGGER.error("Cannot connect %s", error)
+                return self.async_show_form(
+                    step_id="user",
+                    data_schema=DATA_SCHEMA,
+                    errors={"base": "cannot_connect"},
+                )
```

Comment:
Same as above, not related to the PR subject.

Reply:
Will separate PR for these additional checks